src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipChangedEventHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿using NServiceBus;
+using SFA.DAS.ApprenticeCommitments.Jobs.Api;
+using SFA.DAS.ApprenticeCommitments.Jobs.Functions.Infrastructure;
+using SFA.DAS.ApprenticeCommitments.Messages.Events;
+using SFA.DAS.Notifications.Messages.Commands;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs.Functions
+{
+    public class ApprenticeshipChangedEventHandler : IHandleMessages<ApprenticeshipChangedEvent>
+    {
+        private readonly ApplicationSettings settings;
+        private readonly IEcsApi api;
+
+        public ApprenticeshipChangedEventHandler(ApplicationSettings settings, IEcsApi api)
+        {
+            this.api = api;
+            this.settings = settings;
+        }
+
+        public async Task Handle(ApprenticeshipChangedEvent message, IMessageHandlerContext context)
+        {
+            if (!settings.Notifications.Templates.TryGetValue("ApprenticeshipChangedEmail", out var templateId))
+                throw new System.Exception("Missing configuration `Notifications:Templates:ApprenticeshipChangedEmail`");
+
+            var url = $"{settings.ApprenticeCommitmentsWeb.BaseUrl}/Apprenticeships";
+
+            var apprentice = await api.GetApprentice(message.ApprenticeId);
+
+            await context.Send(new SendEmailCommand(
+                templateId,
+                apprentice.Email.ToString(),
+                new Dictionary<string, string>
+                {
+                    { "GivenName", apprentice.FirstName },
+                    { "FamilyName", apprentice.LastName },
+                    { "ConfirmApprenticeshipUrl", url },
+                }));
+        }
```

Comment:
I'm a little confused about this PR. Isn't this email template in the LoginService?  

If they didn't want the MessageHandler calling the API (which I understand), could it have just called teh command layer directly itself. 

We now have emails emanating from two possible sources

Reply:
Yes, these emails originate from the Confirm My Apprenticeship domain, not the login service domain.

The login service sends emails inviting the user to create an account, and to verify their new email address when they want to change it.

CMA sends emails informing the user of a change to their apprenticeship (and potentially in future reminding them they need to sign their apprenticeship after they have created their account)

I don't think it would make sense for the login service to be initiating "your apprenticeship has changed" emails when that has nothing to do with login functionality.  What do you think?